test/dummy/ev3api_motor.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file ev3api_motor.h
+ * @brief モータ関連の列挙体（ダミー）
+ * @author kodama0720, yutotanaka24
+ */
+#pragma once
+
+/**
+ * \brief モータポートを表す番号
```

Comment:
```suggestion
 * @brief モータポートを表す番号
```
コメントの書き方を統一した方がいいかなと思います